src/LineChartStat/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import ReactGA from 'react-ga';
+import firebase from 'firebase/app'
+import LineChart from 'react-linechart';
+import '../../node_modules/react-linechart/dist/styles.css';
+
+class LineChartStat extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            top10 : [
+                //{user: (user object), score: "12"}
+            ]
+        }
+    }
+    
+    componentDidMount() {
+        ReactGA.pageview('/scoreboard');
```

Comment:
This should be updated to reflect the new page.
```suggestion
        ReactGA.pageview('/chart');
```